apple/internal/resource_actions/actool.bzl

```diff
@@ -79,9 +75,10 @@ def _actool_args_for_special_file_types(
         appicon_extension = "stickersiconset"
         icon_files = [f for f in asset_files if ".stickersiconset/" in f.path]
 
-        # TODO(kaipi): We might be processing a resource bundle inside an
-        # ios_extension, in which case the bundle ID might not be appropriate here.
         args.extend([
+            "--include-sticker-content",
+            "--stickers-icon-role",
+            "extension",
```

Comment:
When were these introduced? If after Xcode 14.0, we probably have to conditionalize this.

Reply:
Looks like from Xcode 12 https://github.com/keith/xcode-man-pages/blame/f7483949128973c7f538faa6795f0b0a39c0c1fc/docs/actool.1.html#L274